### PR TITLE
CVSL-471: Fix dockerfile to update before installing make and python

### PR DESCRIPTION
**Pull request by smcveigh941**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,10 +22,12 @@ RUN apt-get update && \
 
 # Stage: build assets
 FROM base as build
-ARG BUILD_NUMBER
-ARG GIT_REF
 
-RUN apt-get install -y make python g++
+ARG BUILD_NUMBER=1_0_0
+ARG GIT_REF=not-available
+
```
Comment (timharrison-moj) [inline Dockerfile:28]: Do these get overwritten with the proper values in the second build stage? Just checking that the dps-monitor will retain the actual Git ref and build version

File: Dockerfile [hunk 2]
```diff
@@ -22,10 +22,12 @@ RUN apt-get update && \
...
+RUN apt-get update && \
+    apt-get install -y make python g++
```
Comment (timharrison-moj) [inline Dockerfile:30]: Why python and g++ here?

**Review by timharrison-moj — APPROVED** [review]
Couple of questions but LGTM

File: Dockerfile (hunk 2 above)
Comment (smcveigh941) [reply Dockerfile:30]: These were here already, unsure what its for... I just added `apt-get update`

File: Dockerfile (hunk 1 above)
Comment (smcveigh941) [reply Dockerfile:28]: Yes seems to be ok, this is copied direct from the template repo

Screenshot 2022-03-24 at 18 54 37

**Pull request merged**